src/rules/font-family-no-duplicate-names/README.md

```diff
@@ -0,0 +1,47 @@
+# font-family-no-duplicate-names
+
+Disallow duplicate font family names.
+
+```css
+a { font-family: "Times New Roman", serif; }
+/**               ↑                 ↑
+ *               These font family names */
+```
+
+This rule checks the `font` and `font-family` properties.
+
+This rule ignores `$sass`, `@less`, and `var(--custom-property)` variable syntaxes.
+
+**Caveat:** This rule does not currently understand escape sequences. If you want to use the font family name "Hawaii 5-0" you will need to wrap it in quotes, instead of escaping it as `Hawaii \35 -0` or `Hawaii\ 5-0`.
```

Comment:
> If you want to use ...

This is inaccurate, right? You do not need to wrap it in quotes to use it. What is the actual consequence related to this rule of not wrapping the name in quotes?